packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -165,13 +153,15 @@ function WPNonProductLineItem( {
 	hasDeleteButton,
 	removeProductFromCart,
 	createUserAndSiteBeforeTransaction,
+	isPwpoUser,
 }: {
 	lineItem: LineItemType;
 	className?: string | null;
 	isSummary?: boolean;
 	hasDeleteButton?: boolean;
 	removeProductFromCart?: () => void;
 	createUserAndSiteBeforeTransaction?: boolean;
+	isPwpoUser?: boolean;
```

Comment:
There's no reason to do anything about it now, but `isPwpoUser` seems business-logic specific enough that I wonder if it would be better to pass around a user flags object instead. With only one flag that would be overkill. But reading this my first thought was "why does the line item component need to know this?", and the answer is that it doesn't, except that the modal copy depends on it. That is, the specificity of `isPwpoUser` is kind of a red herring.